ext/bz2/bz2.c

```diff
@@ -569,9 +569,9 @@ static PHP_FUNCTION(bzdecompress)
 		size = (bzs.total_out_hi32 * (unsigned int) -1) + bzs.total_out_lo32;
 #if !ZEND_ENABLE_ZVAL_LONG64
 		if (UNEXPECTED(size > SIZE_MAX)) {
-			php_error_docref(NULL, E_WARNING, "Decompressed size too big, max is %zd", SIZE_MAX);
+			zend_value_error("Decompressed size too big, max is %zd", SIZE_MAX);
```

Comment:
This one shouldn't be an exception, as it is controlled by the decompressed payload.

Reply:
ACK